driver-core/src/test/functional/com/mongodb/internal/operation/ChangeStreamOperationProseTestSpecification.groovy

```diff
@@ -107,7 +107,10 @@ class ChangeStreamOperationProseTestSpecification extends OperationFunctionalSpe
         waitForLastRelease(async ? getAsyncCluster() : getCluster())
 
         where:
-        async << [true, false]
+        async  | callHasNext
```

Comment:
Added this hack so we can test the path where the getMore fails due to a call to hasNext rather than next

Reply:
Is this missing true, true? If so this would work:

`[async, callHasNext] << [[true, false], [true, false]].combinations()`

Reply:
Ack